gramex/apps/nlg/index.html

```diff
@@ -7,231 +7,317 @@
 </head>
 
 <body>
-    {% from gramex.apps.nlg.nlgapp import read_current_config %}
-    {% set dsid = read_current_config(handler)['dsid'] %}
-    <!-- Include JS dependencies  -->
-    <script src="ui/popper.js/dist/umd/popper.min.js"></script>
-    <script src="ui/bootstrap-select/dist/js/bootstrap-select.min.js"></script>
-    <script src="nlg/nlg.js"></script>
-    <script>
-        // Render the FormHandler table
+  {% set admin_kwargs = handler.kwargs.get('admin_kwargs', '') or {} %}
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="ui/bootstraptheme.css{{ admin_kwargs.get('theme', '') }}">
+  <link rel="stylesheet" href="ui/font-awesome/css/font-awesome.min.css">
+  <script src="ui/popper.js/dist/umd/popper.min.js"></script>
+  <!--IDE HTML-->
+  {% from gramex.apps.nlg.nlgapp import read_current_config, get_user_dir, is_user_authenticated %}
+  {% set dsid = read_current_config(handler).get('dsid') %}
+  <!-- Include JS dependencies  -->
+  {% if not nlg_base %}
```

Comment:
You'll always need jQuery, etc. Mount these under `nlg_base/ui/` explicitly in `nlg/gramex.yaml` -- always.